packages/react-reconciler/src/ReactFiberHooks.js

```diff
@@ -3822,6 +3826,16 @@ if (__DEV__) {
         mountHookTypesDev();
         return mountFormState(action, initialState, permalink);
       };
+    (HooksDispatcherOnMountInDEV: Dispatcher).useActionState =
+      function useActionState<S, P>(
+        action: (Awaited<S>, P) => S,
+        initialState: Awaited<S>,
+        permalink?: string,
+      ): [Awaited<S>, (P) => void, boolean] {
+        currentHookNameInDev = 'useActionState';
+        mountHookTypesDev();
+        return mountFormState(action, initialState, permalink);
```

Comment:
Nit: Let's rename these to "action state" since the form state ones will go away eventually

Reply:
I'm going to do that in a follow up, it was in the original version but the PR was too big, so I'm splitting it out. Wanted to get this reviewed before submitting that to avoid annoying conflicts.